casepro/backend/junebug.py

```diff
@@ -463,3 +467,44 @@ def received_junebug_message(request):
         created_on=datetime.now(), has_labels=True)
 
     return JsonResponse(msg.as_json())
+
+
+@csrf_exempt
+def receive_identity_store_optout(request):
+    '''Handles optout notifications from the Identity Store.'''
+    if request.method != 'POST':
+        return JsonResponse({'reason': 'Method not allowed.'}, status=405)
+
+    try:
+        data = json.loads(request.body)
+    except ValueError as e:
+        return JsonResponse(
+            {'reason': 'JSON decode error', 'details': e.message}, status=400)
+
+    identity_id = data.get('identity_id')
+    optout_type = data.get('optout_type')
+
+    syncer = IdentityStoreContactSyncer()
+
+    # The identity store currently doesn't specify the response format or do
+    # anything with the response.
+
+    # I don't know how to get the org
+    local_contact = syncer.fetch_local(org, identity_id)
```

Comment:
Correct. That is what why I noticed too. I suggest we just declare that
there needs to be exactly one Org for now and that we'll sort out multiple
Orgs in a later sprint.